src/ray/internal/internal.cc

```diff
@@ -38,14 +44,14 @@ std::vector<rpc::ObjectReference> SendInternal(
   TaskOptions options{name, return_num, resources};
 
   char meta_data[3] = {'R', 'A', 'W'};
-  std::shared_ptr<LocalMemoryBuffer> meta =
-      std::make_shared<LocalMemoryBuffer>((uint8_t *)meta_data, 3, true);
+  std::shared_ptr<LocalMemoryBuffer> meta = std::make_shared<LocalMemoryBuffer>(
```

Comment:
The error message is
```
[2025-02-17T16:48:38Z] src/ray/internal/internal.cc:54:45: error: 'reinterpret_cast' from type 'const char*' to type 'uint8_t*' {aka 'unsigned char*'} casts awa
--
  | y qualifiers
  | [2025-02-17T16:48:38Z]    54 \|         std::make_shared<LocalMemoryBuffer>(reinterpret_cast<uint8_t *>(dummy), 13, true);
  | [2025-02-17T16:48:38Z]       \|                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
which means you need to `const_cast` first.

or you could keep the existing code, with `NOLINT` to suppress (I'm ok with it here).

Reply:
Also, `auto meta = `, no need to redeclare type.